ods_ci/utils/scripts/ocm/ocm.py

```diff
@@ -147,7 +147,7 @@ def osd_cluster_create(self):
 
                 version_cmd = f'ocm list versions {chan_grp} | grep -w "{re.escape(version)}*"'
                 versions = execute_command(version_cmd)
-                if versions is not None:
+                if versions:
```

Comment:
Hmm, shouldn't we `log.error()` and return `sys.exit(1)` now, since this case is properly handled? :thinking:

Reply:
Yeah, IIUC, this is about the case when the required version isn't offered by `ocm` command anymore. So we should log some clear error message and return non zero code, isn't so? What happened up to now before your fix? I suppose that the code failed non-clean so we saw this right error right-away. But now it will continue and the empty `version` is provided further on to the `ocm` command which will not make it easier to debug what the hell went wrong, IMO.

Unless I'm missing something. I don't know this part of the code.

Reply:
oh yeah you're right, we definitely should fail the job when there are no corresponding versions in OCM.
I added this logic now.
Thanks for catching that

Reply:
Thanks, LGTM now :)